config/locales/en.yml

```diff
@@ -1182,6 +1182,8 @@ en:
           title: Transfer Samples
           new_project_id: Project to transfer samples to
           submit_button: Submit
+          loading: Loading...
```

Comment:
**nitpick:** Can you update the fr file with these as well please

Reply:
379ffb4f5e83ec280756b0623c1ac6c1cb8229fc